interview

```diff
@@ -0,0 +1,12 @@
+Q1: So you're finishing up your time in this CSCC program. Why did you decide to embark on this journey?
+I want to become a full stack developer as industry is going towards that. Also, I want to learn Java programming which is in great demand.
+
+Q2: I love to travel. Tell me a bit about your favorite travel destination so far. What's on your travel bucket list?
+My favorite travel destination was Bahamas cruise.
+I want to goto Alaska cruise, Europe tour and Banff.
```

Comment:
Great places on your list!  I'm going on a cruise this summer (not Alaska) with our family and the in-laws.  Should be a great time!  I totally want to do Alaska/Banff.  I really want to see some of that wonderful nature!

Reply:
wow nice...Have a nice and safe trip.